### ISSUE TITLE
Whitespace rule should flag spaces around `new`

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3568]
```typescript
const instance =    new      MyClass();
```

That feels morally wrong.

**Comment by ajafff** [comment]
This is basically a duplicate of #3548 and probably many others.
I think you should rather use a formatter than a lint rule.
